core/sms_server_v2.py

```
@@ -0,0 +1,592 @@
+            batch_processor=worker_status,
+        ),
+    )
+
+
+@app.post(
+    "/admin/trigger-recovery",
+    response_model=TriggerRecoveryResponse,
+    tags=["Admin"],
+    summary="Trigger recovery process",
+)
+def trigger_recovery(db: Session = Depends(get_db)) -> TriggerRecoveryResponse:
```

Ruff (0.14.10)

Unused function argument: `db`

(ARG001)